containers/kubernetes/reference-content/multi-az-clusters.mdx

```diff
@@ -0,0 +1,263 @@
+
+* Cluster with an unresponsive Availability Zone
+    <Lightbox src="scaleway-k8s-multi-az-3.webp" size="large" />
+
+```yaml
+data "scaleway_domain_zone" "multi-az" {
+  domain    = "your-domain.tld"
+  subdomain = "scw"
+}
+
+resource "scaleway_domain_record" "multi-az" {
+  dns_zone = data.scaleway_domain_zone.multi-az.id
+  name     = "ingress"
+  type     = "A"
+  data     = kubernetes_service.nginx["fr-par-1"].status.0.load_balancer.0.ingress.0.ip
+  ttl      = 60
+
+  http_service {
+    ips = [
+      kubernetes_service.nginx["fr-par-1"].status.0.load_balancer.0.ingress.0.ip,
+      kubernetes_service.nginx["fr-par-2"].status.0.load_balancer.0.ingress.0.ip,
+    ]
+    must_contain = "up"
+    url          = "http://ingress.scw.yourdomain.tld/up"
+    user_agent   = "scw_dns_healthcheck"
+    strategy     = "all"
+  }
+}
+```
+
+### Storage with VolumeBindingMode
+
+
+In this final section, the focus will be on stateful applications that require persistent volumes within a multi-Availability Zone (AZ) architecture. A prerequisite is having a default `scw-bssd` storage class in your cluster, with the `volumeBindingMode` parameter specifically set to `WaitForFirstConsumer`. 
+You can confirm the presence and settings of your storage classes using the `kubectl` command shown below:
+
+```
+kubectl get storageclasses.storage.k8s.io
+
+NAME                 PROVISIONER        RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+scw-bssd (default)   csi.scaleway.com   Delete          WaitForFirstConsumer   true                   131m
+scw-bssd-retain      csi.scaleway.com   Retain          WaitForFirstConsumer   true                   131m
+```
+<Message type="note">
+  If your existing setup was created with the binding mode `Immediate`, it is necessary to upgrade your cluster to a newer patch version (Kubernetes >=1.24.17, >=1.25.13, >=1.26.8, >=1.27.5, or >=1.28.1). 
+  This upgrade process will automatically modify the storage class to the desired `WaitForFirstConsumer` binding mode.
+</Message>
+
+Using a storage class with `volumeBindingMode` set to `WaitForFirstConsumer` is a requirement when deploying applications across multiple AZs, especially those that rely on persistent volumes. This configuration ensures that volume creation is contingent on the pod's scheduling, aligning with specific AZ prerequisites.
+Creating a volume ahead of this could lead to its arbitrary placement in an AZ, which can cause attachment issues if the pod is subsequently scheduled in a different AZ. YThe `WaitForFirstConsumer` mode ensures that volumes are instantiated in the same AZ as their corresponding node, ensuring distribution across various AZs as pods are allocated. 
```

Comment:
```suggestion
Creating a volume ahead of this could lead to its arbitrary placement in an AZ, which can cause attachment issues if the pod is subsequently scheduled in a different AZ. The `WaitForFirstConsumer` mode ensures that volumes are instantiated in the same AZ as their corresponding node, ensuring distribution across various AZs as pods are allocated. 
```